services/account_type/legacy.yaml

```diff
@@ -0,0 +1,267 @@
+# -*- coding: utf-8 -*-
```

Comment:
Create a new Legacy Template that creates a "super role" that has all of the permissions the current Reactor "connect account" Lambda expects.